powersimdata/scenario/move.py

```diff
@@ -65,27 +60,25 @@ def __init__(self, data_access, scenario_info):
         self._data_access = data_access
         self._scenario_info = scenario_info
```

Comment:
I think you only need the id of the scenario. See previous [comment](https://github.com/Breakthrough-Energy/PowerSimData/pull/520/files?file-filters%5B%5D=.lock&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension&hide-deleted-files=true#r677850959).